src/app.js

```diff
@@ -25,16 +25,13 @@ window.addEventListener('message', function (ev) {
 }, false);
 
 var run = function () {
+  var storage = new Storage(updateFiles);
+
   function loadFiles (files) {
     for (var f in files) {
       var key = utils.fileKey(f);
```

Comment:
Are you referring to something that changed in this Pull Request? Or are you referring to `f` and `key`?